src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -190,6 +196,9 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         this.plagiarismCaseService.getPlagiarismCaseInfoForStudent(this.courseId, this.exerciseId).subscribe((res: HttpResponse<PlagiarismCaseInfo>) => {
             this.plagiarismCaseInfo = res.body ?? undefined;
         });
+        this.sessionService.getCurrentSession(this.exerciseId).subscribe((session: HttpResponse<IrisSession>) => {
+            this.messageStore.dispatch(new SessionIdReceivedAction(session.body!.id));
```

Comment:
https://github.com/ls1intum/Artemis/blob/develop-iris/src/main/java/de/tum/in/www1/artemis/domain/iris/IrisSession.java
seems like the server doesn't return back `id` field, cc @Hialus

Reply:
Works as expected, see {
  "id": 2,
  "messages": [
    {
      "id": 1,
      "sender": "USER",
      "content": [
        {
          "id": 1,
          "textContent": "Hello, world!"
        }
      ]
    }
  ]
}

Reply:
are you sure that the response of `GET programming-exercises/{exerciseId}/sessions` ? It looks like the response of `GET sessions/{sessionId}/messages`

Reply:
You are right, we can discuss this point on Tuesday.

Reply:
> https://github.com/ls1intum/Artemis/blob/develop-iris/src/main/java/de/tum/in/www1/artemis/domain/iris/IrisSession.java seems like the server doesn't return back `id` field, cc @Hialus

The server does return the session id. This is the case for all domain objects.

Reply:
ah so it's implicit 👍 thanks